docs/SETUP.md

```diff
@@ -0,0 +1,107 @@
+# Setup
+
+Make a copy of _.env\_sample_ called _.env_ in the root of the project directory.
+
+From here, you may enter in any relevant information pertaining to your project. A few things that must be decided are:
+
+1. Whether you will use AWS or Azure for storage
+2. SSL Certs and their locations
+3. Where your log files will live
```

Comment:
Please add a quick start section.

Also add section for just running compute worker.